python/cudf_polars/cudf_polars/callback.py

```diff
@@ -183,7 +184,10 @@ def execute_with_cudf(
             nt.set_udf(
                 partial(
                     _callback,
-                    translate_ir(nt),
+                    translate_ir(
+                        nt,
+                        debug_mode=1 if debug_mode else 0,
```

Comment:
Pass a more general config object?

Reply:
We need this to fail before calling `set_udf`, so perhaps something like:

```
ir = translate_ir(nt, ...)
if debug_mode:
    from cudf_polars.dsl.translate import ERRORS
    if has_errors(ERRORS):
         raise ...
nt.set_udf(...)
```
?